Expecto/Expecto.Impl.fs

```diff
@@ -520,6 +535,11 @@ module Impl =
       colour: ColourLevel
       /// Split test names by `.` or `/`
       joinWith: JoinWith
+      // One option is to allow the consumer to provide an activity source
+      // only problem is the only way to update the config is by using the CLIArguments currently
+      // we would have to add a new CLIArgument but that doesn't really work as it's not a reallyCLI option
+      // or have another way of updating the config after it's been created
+      activitySource : ActivitySource option
```

Comment:
As mentioned, we can have an activity source in the config but there's not really a function that lets people update the config.

It might be more tricky to do this option if people are using the `dotnet test` integration so maybe a dedicated source is appropriate.